src/game/boe.graphics.cpp

```diff
@@ -31,7 +31,9 @@
 #include "boe.menus.hpp"
 #include "winutil.hpp"
 #include "prefs.hpp"
+#ifndef MSBUILD_GITREV
 #include "gitrev.hpp"
+#endif
```

Comment:
Just a preprocessor that we set if MSBUILD is handling this. Added a NuGet package called "gitinfo" that makes git status related stuff available to preprocessor.